<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CComponentPromotorDialogComp</class>
 <widget class="QDialog" name="CComponentPromotorDialogComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Promote to another component</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Compatibility to source component</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="InterfacesPolicyCB">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Don't care about interfaces</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Implementing used interfaces</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Implementing all interfaces</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="AttributesPolicyCB">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Don't care about attributes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Containing used attributes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Containing all attributes</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="SubcomponentsPolicyCB">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Don't care about subcomponents</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Containing used subcomponents</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Containing all subcomponents</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Convert to component</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="PackageNameCB"/>
      </item>
      <item>
       <widget class="QComboBox" name="ComponentNameCB"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>CComponentPromotorDialogComp</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CComponentPromotorDialogComp</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
